porcupine/default_keybindings.tcl

```diff
@@ -3,8 +3,10 @@
 # Use Command on mac, Control on other systems
 if {[tk windowingsystem] == "aqua"} {
     set contmand Command
+    event add "<<RightClick>>" <Button-2>
 } else {
     set contmand Control
+    event add "<<RightClick>>" <Button-3>
 }
```

Comment:
`<<TabClosing:ShowMenu>>` is defined below in the same file. Can you delete it and change the `tab_closing` plugin to use `<<RightClick>>` instead?